src/Corax/IndexEntryBuilder.cs

```diff
@@ -30,6 +32,7 @@ public enum IndexEntryFieldType : byte
         
         Tuple = 1 << 2,
         List = 1 << 3,
+        Blittable = 1 << 4,
```

Comment:
In a sense, more than blittable this is `Raw` ... that higher abstraction layers interprete it as a Blittable, it is an entirely different deal.

Reply:
Agree

Reply:
Good idea.

I'm proposing this format for `Raw`
`<type:IndexEntryType><binary_type:Enum><size_of_binary:Int><binary>`

Any thoughts? Anything else might be useful in the future?

Reply:
Not sure if we need `<binary_type:Enum>`. From one side it might be useful in the future, from the other we don't have any other type and I feel that the binary type should be determined at higher level.

@ayende @redknightlois ?

Reply:
Agree. I dont see the need for an enum type. It can be encoded anyways as a int

Reply:
If we need to specify the type of the value, it should be in the parent enum, but `Raw` is enough.
The caller should have the knowledge about that already.